Duckling/Time/CA/Rules.hs

```diff
@@ -1064,7 +1064,7 @@ ruleMorning = Rule
     [ regex "mat(i|í)"
     ]
   , prod = \_ ->
-      let from = hour False 6
+      let from = hour False 4
```

Comment:
why did this change?

Reply:
"matí" means morning, this change is to make the time range consistent with similar languages (ES / PT)